index.js

```diff
@@ -37,20 +38,18 @@ var sourcemapReporter = function(file) {
       colNum = message.col;
     }
 
-    gutil.log(
-      c.red('[') +
-      (
-        typeof message.line !== 'undefined' ?
-          c.yellow( 'L' + lineNum ) +
-          c.red(':') +
-          c.yellow( 'C' + colNum )
-        :
-          c.yellow('GENERAL')
-      ) +
-      c.red('] ') +
-      message.message + ' ' + message.rule.desc + ' (' + message.rule.id + ')');
+    var msgInfo = message.message + ' ' + message.rule.desc + ' (' + message.rule.id + ')';
+
+    var locInfo = c.red('(') + c.yellow(path.relative(process.cwd(), originalPos.source) + ':');
+    if (typeof message.line !== 'undefined') {
+      locInfo += c.yellow(lineNum) + c.red(':') + c.yellow(colNum);
+    } else {
+      locInfo += c.yellow('GENERAL');
+    }
+    locInfo += c.red(')');
+
+    console.log(msgInfo, locInfo, '\n');
```

Comment:
@Kuniwak, I've used `console.log` because the printed timestamp (with `gutil.log`) is too verbose for each error of `csslint`. Moreover I was inspired by https://github.com/sindresorhus/jshint-stylish source code, and I think that we can achieve far better printing format in the future by using `console.log` and https://github.com/sindresorhus/chalk. Anyway, you're the maintainer and if you want absolutely to use `gutil.log` I can change the code in accordance. 
For style issues, line break, etc. please add a coding style guideline `jscsrc.json`. I've autoformated the code with my default coding style config.